tests/Unit/Evolution/DgSubcell/Test_BackgroundGrVars.cpp

```diff
@@ -249,6 +249,9 @@ void test(const gsl::not_null<std::mt19937*> gen, const bool did_rollback) {
   auto box = [&block, &brick, &dg_gr_vars, &element, &element_id,
               &grid_to_inertial_map, &initial_time, &solution,
               &subcell_initial_inertial_coords, &subcell_mesh]() {
+    // Bug in GCC 13.2 where having this class created in-place below causes an
+    // internal compiler error
+    typename subcell_face_gr_variables_tag::type face_gr_var_meta{};
```

Comment:
Fix looks good, but why "meta"?

Reply:
Oh I misread this and thought it was creating the tag, not the type. Should have known since this was used in a box. I'll change it